src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py

```diff
@@ -857,6 +899,10 @@
 helps['vm image accept-terms'] = """
 type: command
 short-summary: Accept Azure Marketplace term so that the image can be used to create VMs
+examples:
+  - name: Accept Azure Marketplace term so that the image can be used to create VMs. (autogenerated)
+    text: az vm image accept-terms --urn publisher:offer:sku:version
```

Comment:
`--urn publisher:offer:sku:version` this is not a real value.

Reply:
Makes sense, we are currently in process of prioritize more concrete parameter values. As per offline discussion, we will still retain such values in the AI as they contain valuable information.